webapp/src/components/expanded_view/component.tsx

```diff
@@ -417,7 +419,7 @@ export default class ExpandedView extends React.PureComponent<Props, State> {
                             </div>
                         </>
                         }
-                        {!isHandRaised && hasReaction && status.reaction &&
+                        {!hasExperimentalFlag() && !isHandRaised && hasReaction && status.reaction &&
```

Comment:
Shouldn't this be `hasExperimentalFlag`?
```suggestion
                        {hasExperimentalFlag() && !isHandRaised && hasReaction && status.reaction &&
```

Reply:
yes, I meant to change it after seeing I got it reversed, but since I was testing multiple things I forgot about this one